src/components/lib.js

```diff
@@ -0,0 +1,8 @@
+import CyverseAnnouncer, { announce } from "./announcer/CyverseAnnouncer";
```

Comment:
I think `CyVerse` should be properly cased, so at least this import should be renamed `CyVerseAnnouncer`, but the file and classes in `CyverseAnnouncer.js` should probably get renamed to `CyVerseAnnouncer` as well.

Reply:
Sounds good. I will rename them.